lib/smart_proxy_dynflow_core/log.rb

```diff
@@ -40,6 +40,11 @@ def log_file
       end
     end
 
+    def initialize(file, *rest)
+      @fd = file.kind_of?(IO) ? file : File.open(file, 'a')
```

Comment:
The `@fd` should be set with `sync = true`, to write the the lines to the file imediatelly